tests/main/prepare-image-grub/task.yaml

```diff
@@ -47,6 +47,7 @@ execute: |
     export SNAPPY_FORCE_SAS_URL=http://$STORE_ADDR
 
     echo Running prepare-image
+    #shellcheck disable=SC2086
```

Comment:
Is this needed? I looked it up in the wiki and it seems to be doing something that is not present in the following line. Also when I run my (18.04) shellcheck against this file I get no complains.